app/common/directives/dropdown.js

```diff
@@ -281,6 +281,29 @@ angular.module('ushahidi.common.dropdown', ['ui.bootstrap.position'])
     };
 })
 
+.directive('isFloating', function () {
```

Comment:
Should change the name of this to `unpositioned` :) This is used to position dropdown menus that are part of a parent container that's not positioned.